### PR TITLE
Transform all <sub>/<sup> from http/*

### PR DESCRIPTION
**Pull request by teoli2003**
This is part of #5012

Replace all \<sub> and \<sup> in http/* by something different (19th, (See xyz), …)

### CODE REVIEW

File: files/en-us/web/http/basics_of_http/evolution_of_http/index.html [hunk 1]
```diff
@@ -22,7 +22,7 @@ <h2 id="Invention_of_the_World_Wide_Web">Invention of the World Wide Web</h2>
  <li>A server to give access to the document, an early version of <em>httpd</em>.</li>
 </ul>
 
-<p>These four building blocks were completed by the end of 1990, and the first servers were already running outside of CERN by early 1991. On August 6<sup>th</sup> 1991, Tim Berners-Lee's <a href="https://www.w3.org/People/Berners-Lee/1991/08/art-6484.txt">post</a> on the public <em>alt.hypertext</em> newsgroup is now considered as the official start of the World Wide Web as a public project.</p>
+<p>These four building blocks were completed by the end of 1990, and the first servers were already running outside of CERN by early 1991. On August 6th> 1991, Tim Berners-Lee's <a href="https://www.w3.org/People/Berners-Lee/1991/08/art-6484.txt">post</a> on the public <em>alt.hypertext</em> newsgroup is now considered as the official start of the World Wide Web as a public project.</p>
```
Comment (hamishwillee) [inline files/en-us/web/http/basics_of_http/evolution_of_http/index.html:25]: You missed cutting the end of the closing tag.

File: files/en-us/web/http/browser_detection_using_the_user_agent/index.html [hunk 2]
```diff
@@ -269,33 +265,33 @@ <h3 id="Rendering_engine">Rendering engine</h3>
 	<tbody>
 		<tr>
 			<td>Gecko</td>
-			<td>Gecko/xyz</td>
+			<td><code>Gecko/xyz<</code>/td>
```
Comment (hamishwillee) [inline files/en-us/web/http/browser_detection_using_the_user_agent/index.html:268]: Looks like tags not quite right

File: files/en-us/web/http/status/308/index.html [hunk 3]
```diff
@@ -24,8 +24,8 @@
   <p><strong>Note:</strong> Some Web applications may use the
     <code>308 Permanent Redirect</code> in a non-standard way and for other purposes. For
     example, Google Drive uses a <code>308 Resume Incomplete</code> response to indicate
-    to the client when an incomplete upload stalled.<sup><a
-        href="https://developers.google.com/drive/v3/web/manage-uploads#resumable">[1]</a></sup>
+    to the client when an incomplete upload stalled. (See <a
+        href="https://developers.google.com/drive/v3/web/manage-uploads#resumable">Perform a resumable downloads</a> on Google Drive documentation.)
```
Comment (hamishwillee) [inline files/en-us/web/http/status/308/index.html:28]: "Perform a resumable downloads" 

TO
> Perform a resumable download 

OR

>  Perform resumable downloads

**Comment by hamishwillee** [comment]
Great thanks. I like your approach. I've tidied a few bits, but can you check the end tag issue above.

File: files/en-us/web/http/basics_of_http/evolution_of_http/index.html (hunk 1 above)
Comment (teoli2003) [reply files/en-us/web/http/basics_of_http/evolution_of_http/index.html]: Thanks!

File: files/en-us/web/http/browser_detection_using_the_user_agent/index.html (hunk 2 above)
Comment (teoli2003) [reply files/en-us/web/http/browser_detection_using_the_user_agent/index.html:268]: Fixed in next commit.

File: files/en-us/web/http/status/308/index.html (hunk 3 above)
Comment (teoli2003) [reply files/en-us/web/http/status/308/index.html:28]: Thanks. Fixed in next commit (went for singular)

**Comment by teoli2003** [comment]
Thanks @hamishwillee for the review! I think I fixed them all. It was my first \<sub> \<sup> removal, so important to validate the approach.

**Review by hamishwillee — APPROVED** [review]
Thanks very much!

**Pull request merged**